### PR TITLE
Pop up

### PR DESCRIPTION
**Pull request by lazyCode-404**
### **In** this branch i implimented

- [ ] basic JavaScript syntax.

- [ ] JavaScript to manipulate DOM elements.

- [ ] JavaScript events.
- [ ]  No linter Errors
- [ ] Used github flow
- [ ] Created Desktop pop up
- [ ] Created mobile pop up

### CODE REVIEW

File: JS/main.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+const cardsinfo = [
+  {
+    reverseCard: 'row',
+    cardName: 'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css', 'html', 'bootstrap', 'ruby'),
+    image: './placeholder.png',
```
Comment (zainsadaqat) [inline JS/main.js]: Please add a link to the source code and live version of the project. 👍🏻

File: JS/main.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  {
+    reverseCard: 'reverse-row',
+    cardName: 'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy tex.`,
+    cardLang: languages('css', 'html', 'bootstrap', 'ruby'),
+    image: './placeholder.png',
+
```
Comment (zainsadaqat) [inline JS/main.js]: Please add a link to the source code and live version of the project. 👍🏻

File: JS/main.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  {
+    reverseCard: 'row',
+    cardName: 'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css', 'html', 'bootstrap', 'ruby'),
+    image: './placeholder.png',
+
```
Comment (zainsadaqat) [inline JS/main.js]: Please add a link to the source code and live version of the project. 👍🏻

File: JS/main.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+  {
+    reverseCard: 'reverse-row',
+    cardName: 'Multi-Post Stories',
+    mainText: `A daily selection of privately personalized reads; no accounts or
+    sign-ups required. has been the industry's standard dummy text
+    ever since the 1500s, when an unknown printer took a standard
+    dummy text.`,
+    cardLang: languages('css', 'html', 'bootstrap', 'ruby'),
+    image: './placeholder.png',
+
```
Comment (zainsadaqat) [inline JS/main.js]: Please add a link to the source code and live version of the project. 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @BwalyaNonde  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] Kindly make the popup window responsive. There's an horizontal scroll bar in the mobile screen. 👍🏻 
Screen Shot 2022-05-19 at 8 08 37 PM

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: JS/main.js (hunk 1 above)
Comment (lazyCode-404) [reply JS/main.js]: done

File: JS/main.js (hunk 4 above)
Comment (lazyCode-404) [reply JS/main.js]: @zainsadaqat 
Thank you very much for your review and helping me become a better developer.
Happy coding!

File: JS/main.js (hunk 3 above)
Comment (zieeco) [reply JS/main.js]: - [x] This issue is well resolved:heavy_check_mark:

File: JS/main.js (hunk 4 above)
Comment (zieeco) [reply JS/main.js]: - [x] This issue is well resolve :heavy_check_mark:

File: JS/main.js (hunk 2 above)
Comment (zieeco) [reply JS/main.js]: - [x] This issue is well resolve :heavy_check_mark:

File: JS/main.js (hunk 1 above)
Comment (zieeco) [reply JS/main.js]: - [x] This issue is well resolve :heavy_check_mark:

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @BwalyaNonde 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Professional README file 💯 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**